crates/settlement-clients/starknet/src/conversion.rs

```diff
@@ -1,9 +1,32 @@
-use starknet::core::types::FieldElement;
+use starknet::core::types::Felt;
 
-pub(crate) fn slice_slice_u8_to_vec_field(slices: &[[u8; 32]]) -> Vec<FieldElement> {
+pub(crate) fn slice_slice_u8_to_vec_field(slices: &[[u8; 32]]) -> Vec<Felt> {
     slices.iter().map(slice_u8_to_field).collect()
 }
 
-pub(crate) fn slice_u8_to_field(slice: &[u8; 32]) -> FieldElement {
-    FieldElement::from_byte_slice_be(slice).expect("could not convert u8 slice to FieldElement")
+pub(crate) fn slice_u8_to_field(slice: &[u8; 32]) -> Felt {
+    Felt::from_bytes_be_slice(slice)
+}
+
+pub(crate) fn u64_from_felt(number: Felt) -> u64 {
```

Comment:
can we make this return a Result instead? if people call this normally in the code, they might not realise that this function can panic